cargo-near/src/commands/new/mod.rs

```diff
@@ -100,6 +102,8 @@ impl NewContext {
             ));
         }
 
+        util::track_usage().await?;
```

Comment:
How is it better to call `async` function here?

I added `#[tokio::main]` directive to the `main` function, but there is still a type issue with `interactive_clap::InteractiveClap` because it's a different type and we have to Futures inside the context

Reply:
Let me walk you through my way of thinking:

1. awaiting for the tracker slows down user for no good reason (imagine I am offline or my internet connection is slow)
2. tracker should not fail the new project creation

That said, we need some background task execution here. See how self-update feature is implemented: https://github.com/near/near-cli-rs/blob/fd3ab99b759b69bb41bd29c1dac7edf8199c2ea5/src/main.rs#L78-L81 - it basically starts a new thread, and then in that thread you can create a new async runtime and run the async function there: https://github.com/near/near-cli-rs/blob/fd3ab99b759b69bb41bd29c1dac7edf8199c2ea5/src/commands/tokens/view_near_balance/mod.rs#L22-L24

Reply:
@garikbesson  please consider using  `let _detached_thread_handle = ...`, unlike `self-update` feature, 
i.e. avoid joining the `track_usage` thread. 
Reason for this :
1. it will simplify implementation
2.  `cargo near new` now does `cargo update`  as one of the steps , and if that request finishes earlier than  `track_usage`,
it means that most likely `track_usage` endpoint is unreachable